quintoandar_kafka/__init__.py

```diff
@@ -1,2 +1,2 @@
 from quintoandar_kafka.idempotence_client import IdempotenceClient  # noqa: F401,E501
-from quintoandar_kafka.kafka_consumer import KafkaConsumerClient  # noqa: F401,E501
+from quintoandar_kafka.kafka_consumer import KafkaDefaultConsumer, KafkaIdempotentConsumer  # noqa: F401,E501
```

Comment:
Alright that's somewhat minor but... what about thinking about a better prefix for open-source tools? I vote for `quinto` :p

i.e. `quintokafka`

And since we already have kafka on the name, no need to repeat it. I suggest renaming the consumer to `consumer`, i. e.:

`quintokafka.consumer`